.gitignore

```diff
@@ -20,6 +20,7 @@ config/ssl
 .env
 .env.old
 .env.production
+.env.dev
```

Comment:
cant we do .env* or something?

Reply:
Removing for now, the thinking here was part of another thought.